scripts/create-stack.sh

```diff
@@ -48,7 +48,7 @@ mkdir -p "logs/$stack_prefix"
 echo "Start ${action_verb} $stack_prefix $stack_type stack"
 ANSIBLE_LOG_PATH=$log_path \
 ANSIBLE_CONFIG=conf/ansible/ansible.cfg \
-  ansible-playbook -v provisioners/ansible/playbooks/"$stack_type".yaml \
+  ansible-playbook -vvv provisioners/ansible/playbooks/"$stack_type".yaml \
```

Comment:
Would this increase of verbosity be polluting the users with too much info? Or is there a specific reason for this?

Reply:
Unintended, ill make a new commit and remove it.